public/app/app.config.js

```diff
@@ -0,0 +1,28 @@
+/*!
```

Comment:
In the open source world, I've always been told that it is advised to add a license header to each file in the repository, as this makes enforcing of the license a lot easier (otherwise you can argue about the origin of a file). Adding a `LICENSE` in the root of your repository is not necessarily sufficient to say that the license applies to all files in the repository.

The minified process should be filtering out these license headers as `/*! */` count as comments as well.